cloudformation/elasticsearch.yaml

```diff
@@ -90,19 +90,21 @@ Resources:
         rules_to_suppress:
           - id: W90
             reason: 'We do not want a VPC for ElasticSearch. We are controlling access to ES using IAM roles'
+    UpdatePolicy:
+      EnableVersionUpgrade: true
     Properties:
       EBSOptions: # Assuming ~100GB storage requirement for PROD; min storage requirement is ~290GB https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/sizing-domains.html
         EBSEnabled: true
         VolumeType: gp2
         VolumeSize: !If [isDev, 10, 73]
       ElasticsearchClusterConfig:
-        InstanceType: !If [isDev, t3.medium.elasticsearch, m5.large.elasticsearch]
+        InstanceType: !If [isDev, c6g.large.elasticsearch, m6g.large.elasticsearch]
```

Comment:
When doing the upgrade was there any downtime?

Reply:
I didn't notice any through my own testing and there weren't any alarms that I noticed through CloudWatch on the ElasticSearch metrics that would indicate service downtime.

Reply:
@rsmayda , to the best of my knowledge it is truly a zero downtime upgrade.

The documentation says so, but I also was curious and deployed the update to my personal stack and left a script constantly querying ES in the background. There were no errors nor data loss during the upgrade. (btw it takes like 40 min to upgrade)

Reply:
That's impressive thanks for checking!